### ISSUE TITLE
QGIS 3.18.1 crash - possible projection problems?

### ISSUE THREAD

**Issue by grantboxer** [qgis/QGIS#42463]
## User Feedback

QGIS crashed on attempted opening of Layer Properties panel. Think is may be due to problems reading the projection of the grid.

## Report Details

**Crash ID**: [b0f4fb2661d5cc6c00b8a904bcdf30bdae3d9d27](https://github.com/qgis/QGIS/search?q=b0f4fb2661d5cc6c00b8a904bcdf30bdae3d9d27&type=Issues)

**Stack Trace**
<pre>
QObject::parent :
QLayout::addChildWidget :
QStackedLayout::insertWidget :
QgsRasterLayerProperties::addPropertiesPageFactory qgsrasterlayerproperties.cpp:561
QgisApp::showLayerProperties qgisapp.cpp:16271
QgisApp::layerProperties qgisapp.cpp:9296
QgisApp::layerTreeViewDoubleClicked qgisapp.cpp:677
QMetaObject::activate :
QAbstractItemView::doubleClicked :
QTreeView::mouseDoubleClickEvent :
QWidget::event :
QFrame::event :
QAbstractItemView::viewportEvent :
QCoreApplicationPrivate::sendThroughObjectEventFilters :
QApplicationPrivate::notify_helper :
QApplication::notify :
QgsApplication::notify qgsapplication.cpp:475
QCoreApplication::notifyInternal2 :
QApplicationPrivate::sendMouseEvent :
QSizePolicy::QSizePolicy :
QSizePolicy::QSizePolicy :
QApplicationPrivate::notify_helper :
QApplication::notify :
QgsApplication::notify qgsapplication.cpp:475
QCoreApplication::notifyInternal2 :
QGuiApplicationPrivate::processMouseEvent :
QWindowSystemInterface::sendWindowSystemEvents :
QEventDispatcherWin32::processEvents :
UserCallWinProcCheckWow :
DispatchMessageWorker :
QEventDispatcherWin32::processEvents :
qt_plugin_query_metadata :
QEventLoop::exec :
QCoreApplication::exec :
main main.cpp:1638
BaseThreadInitThunk :
RtlUserThreadStart :
</pre>

**QGIS Info**
QGIS Version: 3.18.1-Z�rich
QGIS code revision: 202f1bf7e5
Compiled against Qt: 5.11.2
Running against Qt: 5.11.2
Compiled against GDAL: 3.1.4
Running against GDAL: 3.1.4

**System Info**
CPU Type: x86_64
Kernel Type: winnt
Kernel Version: 10.0.19041

**Comment by nyalldawson** [comment]
@grantboxer which plugins do you have installed?

**Comment by gioman** [comment]
> @grantboxer which plugins do you have installed?

@grantboxer or in other words: please try on a new QGIS profile. If the problem persist please list the detailed steps to replicate, attaching sample data if needed.

**Comment by Bananastand-Money** [comment]
I've been getting a similar error to:
https://github.com/qgis/QGIS/issues/41834
https://github.com/qgis/QGIS/issues/41837

I load in the geotiff in my default profile with various plugins but upon selecting "Properties" QGIS would hard crash with no error codes or stack traces. 

I opened a new profile and was able to correctly open properties and began adding my plugins one by one and attempting to re-create the error, however I was unable to re-create the error after adding all of my plugins back. Finally I created a new profile, copied and pasted all the plugins from my default to new profile and everything seems fine.

I've only been able to get it to happen once since starting a new profile and that was after I moved a toolbar, but was then unable to replicate that after installing plugins one by one and moving each individually. I'll keep trying things to see if I can consistently replicate it.

QGIS version
3.18.1-Zürich
QGIS code revision
202f1bf7e5
Compiled against Qt
5.11.2
Running against Qt
5.11.2
Compiled against GDAL/OGR
3.1.4
Running against GDAL/OGR
3.1.4
Compiled against GEOS
3.8.1-CAPI-1.13.3
Running against GEOS
3.8.1-CAPI-1.13.3
Compiled against SQLite
3.29.0
Running against SQLite
3.29.0
PostgreSQL Client Version
11.5
SpatiaLite Version
4.3.0
QWT Version
6.1.3
QScintilla2 Version
2.10.8
Compiled against PROJ
6.3.2
Running against PROJ
Rel. 6.3.2, May 1st, 2020
OS Version
Windows 10 (10.0)
Active python plugins
**db_manager; 
MetaSearch; 
processing; 
slyr_community; 
qProf; 
QGeoloGIS; 
qgis-maptiler-plugin; 
ioGAS; 
geoscience; 
FreehandRasterGeoreferencer; 
enmapboxplugin; 
DataPlotly**
